tests/scripts/kubeadm.sh

```diff
@@ -59,8 +60,13 @@ EOF
 #install k8s node
 install_node(){
     echo "inside install node function"
-    echo "kubeadm join ${1} ${2} ${3} --skip-preflight-checks"
-    sudo kubeadm join ${1} ${2} ${3} --skip-preflight-checks || true
+    if [[ $KUBE_VERSION == v1.5* ]] || [[ $KUBE_VERSION == v1.6* ]] || [[ $KUBE_VERSION == v1.7* ]]; then
+        echo "kubeadm join ${1} ${2} ${3} --skip-preflight-checks"
+        sudo kubeadm join ${1} ${2} ${3} --skip-preflight-checks || true
+    else
+         echo "kubeadm join ${1} ${2} ${3} ${4} ${5} --skip-preflight-checks"
```

Comment:
can you just always pass $(4) and ${5} and not worry about the if/else? If 4 and 5 are blank (for pre-1.7), they should not affect the command, right? that would simplify this PR

Reply:
good, idea sending blank args for 4 and 5.